libtiny_tui/src/tui.rs

```diff
@@ -61,10 +61,13 @@ pub(crate) struct TUI {
     show_statusline: bool,
     /// Is there room for statusline?
     statusline_visible: bool,
+
+	// Do you want to show timestamp in every msg ?
```

Comment:
"Show timestamp in all messages" (which you already used above)

I'd also document the behavior when this is `false` so that the reader won't have to read the code for it, but feel free to leave that to later.